pod-security/baseline/disallow-host-ports/disallow-host-ports.yaml

```diff
@@ -26,7 +26,8 @@ spec:
           spec:
             =(initContainers):
               - =(ports):
-                  - X(hostPort): "null"
+              # the value here is actually not evaluated, the key is the one do not allow.
```

Comment:
Comment here really isn't needed since this has been updated in the documentation.